docs/src/man/interval_hulls.md

```diff
@@ -0,0 +1,104 @@
+# Interval Hulls
+
+In this section we illustrate some interval hull operators
+as well as several plotting functionalities.
+
+```@meta
+DocTestSetup = quote
+    using LazySets, Plots, LazySets.Approximations
+end
+```
+
+## Balls and singleton plots
+
+Consider a ball in the 2-norm:
+
+```@example
+X = Ball2(ones(2), 0.5)
+```
+Observe that, by default, the coefficients of this set are 64-bit floating point
+numbers.
+
+We can plot this set using the `plot` function. Recall that, by design, lazy sets
+plots overapproximate with box directions only. To have a sharp definition of the
+borders, use the accuracy as a second argument:
+
+```julia
+plot(X, 1e-3, aspectratio=1)
+```
+
+Let's add to this plot some points of the set which are farthest in some given directions.
+Single points can be plotted using the `Singleton` type.
+
+To add a new plot to the previous one, use `plot!`. In the third line we plot the
+center of the ball with a cross marker.
+
+!!! note
+    To see the list of available plot keyword arguments, use the `plotattr([attr])`
+    function, where `attr` is the symbol `:Plot`, `:Series`, `:Axis` or `:Subplot`.
+
+```julia
+plot!(Singleton(σ([1., 0], X)))
+plot!(Singleton(σ([1., 1], X)))
+plot!(Singleton(X.center), markershape=:x)
+```
+
+## Symmetric interval hull
+
+The symmetric interval hull is defined in the `Approximations` module:
+
+```julia
+julia> S = Approximations.symmetric_interval_hull(X)
```

Comment:
No need to write `Approximations.` since you included it above. I would make it consistent: either include it visibly for the user once and omit it then, or write it out all the time, but then do not include it above.

Reply:
ok. unfortunately, it seems that the `@meta` environment affects doctests but not `@example` blocks..